src/Entity/Int32EntityId.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+namespace Wikibase\DataModel\Entity;
+
+use InvalidArgumentException;
+
+/**
+ * Interface for EntityIds that can be converted to a unique positive 32 bit integer in the range
+ * [1..2147483647], and back from the entity type and the number. Entity types that do not meet this
```

Comment:
There is no unsigned integer in PHP, not in the 32 bit version and not in the 64 bit version. Calling this "31 bit" when the whole world calls this concept 32 bit is doing nothing but adding confusion.